client/rpcserver/handlers.go

```diff
@@ -448,15 +448,45 @@ func handleOrderBook(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
 
 // parseCoreOrder converts a *core.Order into a *myOrder.
 func parseCoreOrder(co *core.Order, b, q uint32) *myOrder {
-	// settled calculates how much of the order has been finalized.
-	settled := func(qty uint64, matches []*core.Match) (settled uint64) {
-		for _, match := range matches {
-			if (match.Side == order.Maker && match.Status >= order.MakerRedeemed) ||
-				(match.Side == order.Taker && match.Status >= order.MatchComplete) {
-				settled += match.Qty
+	// matchesParser parses core.Match slice & calculates how much of the order
+	// has been settled/finalized.
+	matchesParser := func(matches []*core.Match) (ms []match, settled uint64) {
+		ms = make([]match, 0, len(matches))
+		for _, m := range matches {
+			// Sum up settled value
+			if (m.Side == order.Maker && m.Status >= order.MakerRedeemed) ||
+				(m.Side == order.Taker && m.Status >= order.MatchComplete) {
+				settled += m.Qty
 			}
+			match := match{
+				MatchID:  m.MatchID.String(),
+				Status:   m.Status.String(),
+				Revoked:  m.Revoked,
+				Rate:     m.Rate,
+				Qty:      m.Qty,
+				Side:     m.Side.String(),
+				FeeRate:  m.FeeRate,
+				Stamp:    m.Stamp,
+				IsCancel: m.IsCancel,
```

Comment:
I believe IsCancel is always false as they don't show up anyway.

Reply:
Oh, should I return always false ? or maybe get rid of the field ?

Reply:
I dunno, I guess it's fine like it is. Maybe one day there will be a true?